server/flaskr/utils/input_validation.py

```
@@ -21,53 +32,95 @@ def verify_missing_inputs(data: dict, validators: list[InputValidator]) -> list[
 # TODO: Test it :)
 class UserIdValidator(_ObjectIdValidator):
     dataHandler = UserDataHandler
 
+
 class ProjectIdValidator(_ObjectIdValidator):
     dataHandler = ProjectDataHandler
 
+
 # TODO: Test it :)
 class EnumValidator(InputValidator):
     def __init__(self, field_name, enum):
```

Ruff (0.14.14)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)